internal/cli/profile_check.go

```diff
@@ -32,25 +33,44 @@ var (
 		RunE:  runProfileCheck,
 	}
 
-	flagProfileCheckName      string
-	flagProfileCheckPath      string
-	flagProfileCheckOsVersion string
+	flagProfileCheckName           string
+	flagProfileCheckPath           string
+	flagProfileCheckSkipRemoteless string
```

Comment:
the double-negative here is confusing. Should this be `SkipRemote`?

Reply:
This is a flag for `torcx profile check` (when set to true, default is false) means "please ignore any image, in the profile being checked, that does not specify a remote". This is to avoid blocking on images downloaded by other tools (e.g. tectonic-torcx) or manually. Any better suggestion on how to name it?

Reply:
something like `only-remote` ?

Reply:
Ack.